src/cli/load.js

```diff
@@ -220,22 +234,42 @@ export const makeLoadDefaultPlugins = (
     }
     addToRepoIdRegistry(options.output);
     const graph = await saveGraph(defaultAdapterLoaders(), options.output);
-    await saveCred(graph, options.output);
+    await saveCred(graph, options.output, options.weightsPath || null);
```

Comment:
if we make it an optional arg (see above) then we won't need the `|| null`

Reply:
Changed it.